packages/backend/README.md

```diff
@@ -0,0 +1,40 @@
+![GitHub license](https://img.shields.io/badge/license-Apache_2.0-blue.svg)
+
+# Application Wizard
+
+![](screenshot.png)
+
+With the Application Wizard extension, you can benefit from a rich user experience for yeoman generators. This extension allows developers to reuse existing yeoman generators and provide wizard-like experience with no development efforts.
+
+## Requirements
```

Comment:
it seems you need **both** Theia and VSCode from this section

Reply:
Why? This is with or.
- [VSCode](https://code.visualstudio.com/) 1.39.2 or higher or [Theia](https://www.theia-ide.org/) 0.12 or higher.